.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: format
+      run: cargo fmt
+    - name: Build
+      run: cargo build --verbose
+   
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)